src/layout/grid/gridColumn.jsx

```diff
@@ -128,6 +128,8 @@ const propTypes = {
      * Deprecated prop. Please use `classes` to override styles.
      */
     width: PropTypes.oneOf(['auto', true, false, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]),
+
+    responsiveQueryType: PropTypes.oneOf(['media', 'container']),
```

Comment:
**NAB**: Would be great, if we can add a single line description indicating what this prop would do?

Reply:
Done!